packages/jest-fake-timers/src/legacyFakeTimers.ts

```diff
@@ -29,11 +29,13 @@ type Timer = {
 };
 
 type TimerAPI = {
+  cancelAnimationFrame: typeof window.cancelAnimationFrame | undefined;
```

Comment:
I'd just type it out manually rather than fetching it from `window`

Reply:
done in [`806e5fa` (#11567)](https://github.com/facebook/jest/pull/11567/commits/806e5fa5c22a9783b4e7f3eb150c3bf4638eaa15)